ui/src/components/EntitySearch/EntitySearchResultsRow.jsx

```diff
@@ -75,9 +75,11 @@ class EntitySearchResultsRow extends Component {
     const isPrefix = !!highlights.length;
     const resultClass = c('EntitySearchResultsRow', 'nowrap', { active: isActive }, { prefix: isPrefix });
     const highlightsClass = c('EntitySearchResultsRow', { active: isActive });
+    const viewed = entity.lastViewed ? 'viewed' : '';
+
     return (
       <>
-        <tr key={entity.id} className={resultClass}>
+        <tr key={entity.id} className={`${resultClass} ${viewed}`}>
```

Comment:
Here you could maybe use the classnames library for class name composition, something like this?:
`className={c(resultClass, { viewed: !!entity.lastViewed })} `